bq-metrics-extractor/config/config.yaml

```diff
@@ -0,0 +1,20 @@
+logging:
+  level: INFO
+  loggers:
+    org.ostelco: DEBUG
+  appenders:
+    - type: console
+      layout:
+        type: json
+        customFieldNames:
+           level: severity
+
+bqmetrics:
+  - type: summary
+    name: active_users
+    help: Number of active users
+    resultColumn: count
+    sql: >
+        SELECT count(distinct user_pseudo_id) AS count FROM `pantel-2decb.analytics_160712959.events_*`
```

Comment:
As of now, the sql in config is static.
We need to make it as a template.
Table name to be changed from `events_*` to `events_${yyyyMMdd}`

Here, the `suffix` is yesterday’s date in `yyyyMMdd` format. (edited)
There are some libraries which we can use, which enable `freemarker` expressions in `dropwizard config` file

Reply:
Moved this comment to TODO, didn't fix it properly.